HealthcareSystem/GraphicalEditor/MainWindow.xaml.cs

```diff
@@ -72,35 +79,82 @@ public MapObjectEntity DisplayMapObject
             }
         }
 
-        public MainWindow()
+        private MapObject _selectedMapObject;
+
+        public MapObject SelectedMapObject
         {
+            get
+            {
+                return _selectedMapObject;
+            }
+            set
+            {
+                _selectedMapObject = value;
```

Comment:
Is this property used for bindings? Why doesn't it use OnPropertyChanged? If it's not used for bindings, why a property and not just a field?

Reply:
i didn't write that too

Reply:
These + + + + said opposite.